src/ralph/sim_cards/models.py

```diff
@@ -0,0 +1,120 @@
+from django.conf import settings
+from django.core.validators import (
+    MaxLengthValidator,
+    MinLengthValidator,
+    RegexValidator
+)
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from ralph.back_office.models import BackOfficeAssetStatus, Warehouse
+from ralph.lib.mixins.models import (
+    AdminAbsoluteUrlMixin,
+    NamedMixin,
+    TimeStampMixin
+)
+from ralph.lib.transitions.fields import TransitionField
+
+
+class CellularCarrier(AdminAbsoluteUrlMixin, NamedMixin, models.Model):
+    pass
+
+
+class SIMCard(AdminAbsoluteUrlMixin, TimeStampMixin, models.Model):
+    pin1 = models.CharField(
+        max_length=8, null=True, blank=True,
+        help_text=_('Required numeric characters only.'),
+        validators=[
+            MinLengthValidator(1),
+            MaxLengthValidator(8),
+            RegexValidator(
+                regex='^\d+$',
```

Comment:
Don't Repeat Yourself - please move ``RegexValidator(regexp='^\d+$', ..)`` to a variable